### ISSUE TITLE
Stuck on connecting when no webcam is given

### ISSUE THREAD

**Issue by matcool** [ianramzy/decentralized-video-chat#36]
Would be nice to use this for just a regular call or screen sharing

**Comment by Chaphasilor** [comment]
yes, we are aware of the issue and this will probably be changed when fixing #10

**Comment by summersab** [comment]
Adding a little +1 to this, I think it would be nice to have checkboxes/toggles to specify which devices to share instead of requiring all of them. As mentioned by @matcool, it would be nice to use it for just audio instead of requiring access to the webcam.

Awesome tool, though!

**Comment by watthem** [comment]
Another +1 from me - I'd love to try this service but my computer doesn't currently have a webcam. It would be nice if you were given an option to share your screen instead when there is not a camera.
